modules/acquia_cms_headless/tests/src/Functional/DashboardTestBase.php

```diff
@@ -35,7 +32,13 @@ protected function setUp(): void {
       $this->markTestSkipped('This test cannot run in an Acquia Cloud IDE.');
     }
     parent::setUp();
-    $this->visitHeadlessDashboard();
+    $account = $this->drupalCreateUser();
+    $account->addRole('headless');
```

Comment:
under**visitHeadlessDashboardAdmin**  administrator role is mentioned rest everything is same.
@rajeshreeputra Is this correct or do we need to make some changes?

cc: @vishalkhode1

Reply:
this is correct.